test/unit/user_test.rb

```diff
@@ -206,10 +206,6 @@ class UserTest < ActiveSupport::TestCase
       @rubygems.first.versions.first.update! indexed: false
       assert_equal 2, @user.rubygems_downloaded.count
     end
-
-    should "total their number of pushed rubygems" do
-      assert_equal @user.total_rubygems_count, 3
-    end
```

Comment:
@sferik I ended up with  rubygems_download method is not suitable for this case, because it sorts the results which is unnecessary for just count. I'll convert it again to 'total_rubygems_count' method with the test you mentioned.